lmdeploy/cli/entrypoint.py

```diff
@@ -23,6 +24,14 @@ def run():
     args = parser.parse_args()
 
     if 'run' in dir(args):
+        from lmdeploy.utils import get_model
+        model_path = getattr(args, 'model_path', None)
+        if model_path is not None and not os.path.exists(args.model_path):
+            args.model_path = get_model(args.model_path)
+        model_path_or_server = getattr(args, 'model_path_or_server', None)
+        if model_path_or_server is not None and ':' not in model_path_or_server:  # noqa: E501
```

Comment:
and have to check `model_path_or_server` not exists.
maybe we could update `get_model`  and let it deal with this case. eg:  directly return `model_path` if it exists
https://github.com/InternLM/lmdeploy/blob/2ed7ca6cb5674c5c88a9c07ba89592426e418971/lmdeploy/utils.py#L196-L202

Reply:
I don't think `get_model` should handle the server address

Reply:
I do not mean `get_model` need to handle the server address but the existing local path.
Currently, if the input of `get_model` is a local model path with relative path torCWD, it will still download the ckpt. That's why each time calling `get_model`, need to check and make sure the input path not exist.

```
if model_path is not None and not os.path.exists(args.model_path):
            args.model_path = get_model(args.model_path)
```

For this case, `model_path_or_server` is not checked whether it exists before feed into `get_model`.

Reply:
Let's minimize the change for this version
Try to handle it in CLI. Leave the `get_model` update to the next version. 
Because several modules call `get_model`, I don't want this PR to bring any side effects now. 
We are at the edge of releasing v0.4.0

Reply:
ok. Then, could  update to 
```
        if model_path_or_server is not None and ':' not in model_path_or_server and not os.path.exists(model_path_or_server):  # noqa: E501
            args.model_path_or_server = get_model(args.model_path_or_server)
```